quarkus-micrometer does not work with quarkus-kafka-streams

**Describe the bug**
Micrometer extension does not work with quarks-kafka-streams extension.

**Expected behavior**
Micrometer extension works with Kafka streams

**Actual behavior**
`./gradlew quarkusDev` fails with a `java.lang.LinkageError`:
```
> Task :quarkusDev
Caching disabled for task ':quarkusDev' because:
  Build cache is disabled
Task ':quarkusDev' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Starting process 'command '/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home/bin/java''. Working directory: /Users/mccaig/Downloads/reproducer/build/classes/java/main Command: /Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home/bin/java -Xdebug -Xrunjdwp:transport=dt_socket,address=0.0.0.0:5005,server=y,suspend=n -XX:TieredStopAtLevel=1 -Xverify:none -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dquarkus-internal.serialized-app-model.path=/Users/mccaig/Downloads/reproducer/build/tmp/quarkusDev/quarkus-app-model.dat -jar /Users/mccaig/Downloads/reproducer/build/code-with-quarkus-1.0.0-SNAPSHOT-dev.jar
Successfully started process 'command '/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home/bin/java''
Listening for transport dt_socket at address: 5005
2020-09-30 16:30:12,629 INFO  [org.jbo.threads] (main) JBoss Threads version 3.1.1.Final
2020-09-30 16:30:13,195 INFO  [io.qua.dep.dev.IsolatedDevModeMain] (main) Attempting to start hot replacement endpoint to recover from previous Quarkus startup failure
2020-09-30 16:30:13,195 ERROR [io.qua.run.boo.StartupActionImpl] (Quarkus Main Thread) Error running Quarkus: java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at io.quarkus.runner.bootstrap.StartupActionImpl$3.run(StartupActionImpl.java:134)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ExceptionInInitializerError
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at java.base/java.lang.Class.newInstance(Class.java:584)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:60)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:38)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:106)
        at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:29)
        ... 6 more
Caused by: java.lang.RuntimeException: Failed to start quarkus
        at io.quarkus.runner.ApplicationImpl.<clinit>(ApplicationImpl.zig:233)
        ... 15 more
Caused by: java.lang.LinkageError: loader constraint violation: loader 'app' wants to load class io.quarkus.runtime.RuntimeValue. A different class with the same name was previously loaded by io.quarkus.bootstrap.classloading.QuarkusClassLoader @5bd03f44. (io.quarkus.runtime.RuntimeValue is in unnamed module of loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @5bd03f44, parent loader 'platform')
        at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1017)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:800)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:698)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:621)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:579)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at io.quarkus.resteasy.common.runtime.ResteasyInjectorFactoryRecorder.setup(ResteasyInjectorFactoryRecorder.java:27)
        at io.quarkus.deployment.steps.ResteasyCommonProcessor$setupResteasyInjection-1799175235.deploy_0(ResteasyCommonProcessor$setupResteasyInjection-1799175235.zig:124)
        at io.quarkus.deployment.steps.ResteasyCommonProcessor$setupResteasyInjection-1799175235.deploy(ResteasyCommonProcessor$setupResteasyInjection-1799175235.zig:40)
        at io.quarkus.runner.ApplicationImpl.<clinit>(ApplicationImpl.zig:213)
        ... 15 more
```


**To Reproduce**

Steps to reproduce the behavior:
1. Extract [reproducer.zip](https://github.com/quarkusio/quarkus/files/5308730/reproducer.zip)
2. Run `./gradlew quarkusDev`
3. Observe error

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: `Darwin mccaig-macOS 19.6.0 Darwin Kernel Version 19.6.0: Thu Jun 18 20:49:00 PDT 2020; root:xnu-6153.141.1~1/RELEASE_X86_64 x86_64`
 - Output of `java -version`: 
```
openjdk version "11.0.8" 2020-07-14 LTS
OpenJDK Runtime Environment Zulu11.41+23-CA (build 11.0.8+10-LTS)
OpenJDK 64-Bit Server VM Zulu11.41+23-CA (build 11.0.8+10-LTS, mixed mode)
```
 - GraalVM version (if different from Java): N/A
 - Quarkus version or git rev: `1.8.1.Final`
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): `Gradle 6.4.1`

--- a/extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/binder/VertxBinderProcessor.java
+++ b/extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/binder/VertxBinderProcessor.java
@@ -34,6 +34,9 @@ public class VertxBinderProcessor {
         }
     }
 
+    // avoid imports due to related deps not being there
+    static final String VERTX_CONTAINER_FILTER_CLASS_NAME = "io.quarkus.micrometer.runtime.binder.vertx.VertxMeterBinderContainerFilter";
+
     @BuildStep(onlyIf = { VertxBinderEnabled.class })
     ResteasyJaxrsProviderBuildItem enableResteasySupport(Capabilities capabilities,
             BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
@@ -45,7 +48,7 @@ public class VertxBinderProcessor {
                 .addBeanClass(VertxMeterBinderContainerFilter.class)
                 .setUnremovable().build());
 
-        return new ResteasyJaxrsProviderBuildItem(VertxMeterBinderContainerFilter.class.getName());
+        return new ResteasyJaxrsProviderBuildItem(VERTX_CONTAINER_FILTER_CLASS_NAME);
     }
 
     @BuildStep(onlyIf = VertxBinderEnabled.class)

